ldi-extensions/ldes-discoverer/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/discoverer/config/LdesDiscovererConfig.java

```diff
@@ -5,14 +5,16 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
+import java.util.Map;
+
 @Configuration
 @ConfigurationProperties
 public class LdesDiscovererConfig {
 	public static final String DEFAULT_RDF_FORMAT = "text/turtle";
 	private String url;
 	private String sourceFormat;
 	private String outputFormat;
-
+	private Map<String, String> requester;
```

Comment:
i see this here, but nowhere in docs. For me, it isn't clear what this map entails.. in context of this PR i presume request executor config?

Reply:
At first, I used a map, but then I decided to change this and I must have forgotten to remove this as well